org.eclipse.jdt.core.tests.compiler/src/org/eclipse/jdt/core/tests/compiler/regression/LookupTest.java

```diff
@@ -1921,8 +1921,8 @@ public void test059() {
 		"Zork cannot be resolved to a type\n" +
 		"----------\n" +
 		"2. ERROR in X.java (at line 10)\n" +
-		"	this.bb();\n" +
-		"	     ^^\n" +
+		"	Object o = this.bb();\n" +
+		"	                ^^\n" +
```

Comment:
The assignment to Object o I presume is to force this call to be in non-vanilla context so error message can be maintained. I assume there are other tests that where the absence of the message is the expected outcome as that would be the "fix."

Reply:
> The assignment to Object o I presume is to force this call to be in non-vanilla context so error message can be maintained. 

correct

> I assume there are other tests that where the absence of the message is the expected outcome as that would be the "fix."

Not quite sure I understand. I did change a few tests to expect fewer errors. But for a direct opposite just see ProblemTypeAndMethodTest.testMissingClass_returnType_OK()

Would you prefer the above test source to remain unchanged and the expectation changed instead?

Reply:
> Not quite sure I understand. I did change a few tests to expect fewer errors. But for a direct opposite just see ProblemTypeAndMethodTest.testMissingClass_returnType_OK()
> 
> Would you prefer the above test source to remain unchanged and the expectation changed instead?

No, I am ok with it as is since ProblemTypeAndMethodTest.testMissingClass_returnType_OK() covers the other scenario.